test-server/fixtures/asset-paths/css-urls.html

```diff
@@ -15,5 +15,7 @@
     <div class="with-rewritten-background-img"></div>
     <div class="with-rewritten-background-img-inline"></div>
     <div style="background: url('/img?url=fixtures/pink.png'); no-repeat center;"></div>
+    <div style="background: url('https://www.chromatic.com\emails\icon-chroma-64x64.png?test=query'); no-repeat center;"></div>
+    <img src="\img\another\no-content-type\first"/>
```

Comment:
Is your fix only changing asset paths in CSS `url()`s? I don't think this image tag will be processed by that. This should be tested with a css property like the others in this file.